OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -25,27 +25,39 @@
     <Reference Include="Eluant, Version=1.0.5229.27703, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\thirdparty\Eluant.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
Yep (read: i'll take your word on it, since VS won't run on my system).  This illustrates why these project files are so terrible to work with.  I really must commend microsoft (and xamarin for their parser) for creating a build system that is both more opaque and less cross-platform compatible than standard makefiles - that's an impressive technical feat ;)